db/db.pl

```diff
@@ -1181,9 +1288,9 @@ sub usersCreate
   }
 }';
 
-    print "Creating users_v5 index\n" if ($verbose > 0);
-    esPut("/${PREFIX}users_v5", $settings);
-    esAlias("add", "users_v5", "users");
+    print "Creating users_v6 index\n" if ($verbose > 0);
```

Comment:
Actually when adding a field, you don't need to increment version number.  This should stay v5